code/style.css

```diff
@@ -1,5 +1,35 @@
 /* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
 
+
 body {
-  background-color: lightblue;
+  background-color: rgba(145, 213, 147, 0.646);
+}
+
+.main-image {
```

Comment:
To make this image not be distorted, you can try adding this to the styling of it:
`object-fit: cover;`
That will crop the image as a passe-partout ⭐

Reply:
Another idea is to add some opacity to the image, like this:
`opacity: 0.7;`

and then add absolute positioning to your heading - to create a hero image. Note - this is just a suggestion for you to play around with:
```
h1 {
  font-size: 75px;
  font-weight: bolder;
  font-family: Arial, Helvetica, sans-serif;
  color: white;
  position: absolute;
  top: 200px;
  left: 30%;
  text-shadow: 4px 4px goldenrod;
}
```